Answers.md

```diff
@@ -1,5 +1,21 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+    They're just what they sound like, really; client side routing is done on the user's computer, server side routing is done at the server.
+    Client side routing allows us a far greater economy of bandwidth, better responsiveness from the perspective of the user,
+    and the ability to do fun stuff with routes and matches.   
+
```

Comment:
Pretty much. Client side routing is where the url is update and all the changes to the page are done on the client side.

Server side routing is where the server actually downloads and completely new page and then passes it back to the client for rendering